clippy_utils/src/lib.rs

```diff
@@ -135,6 +135,11 @@ macro_rules! extract_msrv_attr {
     };
 }
 
+// Returns true if the given item behind the `def_id` is accessible to other crates.
+pub fn is_item_exported(cx: &LateContext<'_>, def_id: LocalDefId) -> bool {
+    cx.access_levels.is_exported(def_id)
+}
```

Comment:
What's wrong with `cx.access_levels.is_exported`?

Reply:
I thought it would be more readable... but it's not really...

I'll remove that function again :upside_down_face: